api/utils.py

```
@@ -1,12 +1,81 @@
+) -> Dict[str, Dict[str, Any]]:
+    """
+    Create a combined description from a dictionary of table descriptions.
+
+    Args:
+        table_info (Dict[str, Dict[str, Any]]): Mapping of table names to their metadata.
+        batch_size (int): Number of tables to process per batch when calling the LLM (default: 10).
+    Returns:
+        Dict[str, Dict[str, Any]]: Updated mapping containing descriptions.
+    """
+    if not isinstance(table_info, dict):
+        raise TypeError("table_info must be a dictionary keyed by table name.")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)